projects/wave-core/src/lib/users/user.service.ts

```diff
@@ -516,6 +521,8 @@ export class UserService {
             } as MappingTransform,
             coords: coords as { crs: string, origin: number[], scale: number[], size: number[] },
             provenance: channelProvenance,
+            time_start: !!channel.time_start ? new TimePoint(channel.time_start) : new TimePoint('') ,
+            time_end: !!channel.time_end ? new TimePoint(channel.time_end) : new TimePoint('') ,
```

Comment:
Same here.

Reply:
fixed in 6da1d4c74b7a75a702074ebf5ef83c4bda092d4f